addons/stock/models/stock_package_level.py

```diff
@@ -142,7 +144,16 @@ def _generate_moves(self):
 
     @api.model
     def create(self, vals):
-        result = super(StockPackageLevel, self).create(vals)
+        _logger.info('now in package level create  %s' % vals)
+        print('now in package level create  %s' % vals)
+        try:
+            result = super(StockPackageLevel, self).create(vals)
+        except Exception:
+            _logger.info('failed create' % vals)
+        if result:
+            _logger.info(' package level create WAS SUCCESSFUL')
+            print(' package level create WAS SUCCESSFUL  ')
```

Comment:
Should this be a PR to Odoo or just a branch that is used in situations that require debug?